include/swift/SIL/SILDeclRef.h

```diff
@@ -361,6 +361,13 @@ struct SILDeclRef {
     return r;
   }
 
+  /// Returns this `SILDeclRef` with the `loc` replaced with `decl`.
```

Comment:
Should this be SWIFT_ENABLE_TENSORFLOW?

Reply:
The pieces above it are tensorflow too, and they have a SWIFT_ENABLE_TENSORFLOW.